### DISCUSSION TITLE
Persian date picker

### DISCUSSION THREAD

**Discussion by Hassan198766** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3721] [Q&A]
hello again 
I request the creators of this excellent program to add the Persian calendar to the parts of the program as Persian date picker if possible. The format of Persian calendar is : "yyyy/mm/dd" 
tanks a lot

**Comment by corvinsz** [comment]
@Hassan198766 is your problem related to this issue #3644?
As @nicolaihenriksen pointed out in his [comment](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/3644#issuecomment-2402023270) this could very well be a WPF problem.
I also can't judge whether this is a WPF-nativ "bug" or not, and the person that opened the issue never answered.

**Reply by Hassan198766** [reply]
tanks for your responding - I resolve this issue by add custom date picker(PersianDatePicker)  inside the textbox  of material design
